src/components/Table/Internal/octable.module.scss

```diff
@@ -171,8 +171,6 @@
 
         &-fix-right-first,
         &-fix-right-last {
-            box-shadow: -1px 0 0 $table-border-color;
```

Comment:
This should be configurable via props. The actions column in Succession Planning needs a drop shadow. We'll also need to determine whether to only show actions on hover, etc.